src/components/UI/FetchError/index.tsx

```diff
@@ -5,6 +5,20 @@ const refreshPage = (): void => {
   window.location.reload();
 };
 
+const getHumanReadableErrorMessages = (error: ErrorFromFetch): string => {
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore
+  const errorStatusCode = error.error.response.status;
```

Comment:
Would it make sense to put the url on the message too?.It would make debugging errors